src/sonic-eventd/tools/rsyslog_volume_test.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import subprocess
+import time
+import logging
+import argparse
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    handlers = [
+        logging.FileHandler("debug.log"),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+def read_events_from_file(count):
+    logging.info("Reading from file generated by events_tool")
+    lines = 0
+    with open('STDOUT', 'r') as file:
+        lines = file.readlines()
+        logging.info("Should receive {} events and got {} events\n".format(count, len(lines)))
+    assert len(lines) == count
+
+def start_tool():
+    logging.info("Starting events_tool\n")
+    proc = subprocess.Popen(["./events_tool", "-r"])
+    return proc
+
+def run_test(file, count):
+    # log messages to see if events have been received
+    tool_proc = start_tool()
+
+    time.sleep(2) # buffer for events_tool to startup
+
+    log_messages_cnt = 0
+    logging.info("Generating logger messages\n")
```

Comment:
Can we have a default behavior with no file?  -- like sending few hardcoded events 10 or 15
Also can we default count to a higher value, like 1000

Reply:
Done